lms/djangoapps/certificates/admin.py

```diff
@@ -46,8 +47,16 @@ class CertificateTemplateAssetAdmin(admin.ModelAdmin):
     list_display = ('description', '__unicode__')
 
 
+class GeneratedCertificateAdmin(admin.ModelAdmin):
+    """
+    Django admin customizations for GeneratedCertificae model
+    """
+    list_display = ('id', 'course_id', 'mode', 'user')
```

Comment:
Those to me were the fields that uniquely identified a cert.  Course, Mode, and User.  All other fields are visible and editable on a per record basis, but this seemed clean for what to show in the list view.  We can certainly add more fields if people feel it necessary.